### ISSUE TITLE
CLJS warning: abs already refers to: cljs.core/abs being replaced by: quil.core/abs

### ISSUE THREAD

**Issue by p-himik** [quil/quil#374]
At least when using shadow-cljs, compiling a sketch with quil results in:
```clj
WARNING: abs already refers to: #'clojure.core/abs in namespace: quil.core, being replaced by: #'quil.core/abs

------ WARNING #1 - :redef -----------------------------------------------------
 Resource: quil/core.cljc:258:1
--------------------------------------------------------------------------------
 255 |      [shader uniform-name data]
 256 |      (.setUniform shader uniform-name data)))
 257 | 
 258 | (defn
-------^------------------------------------------------------------------------
 abs already refers to: cljs.core/abs being replaced by: quil.core/abs
--------------------------------------------------------------------------------
 259 |   ^{:requires-bindings false
 260 |     :processing-name "abs()"
 261 |     :p5js-name "abs()"
 262 |     :category "Math"
--------------------------------------------------------------------------------
```

**Comment by dgtized** [comment]
This is fixed in the 4.3-SNAPSHOT release, will close once it's on a versioned release.

**Comment by dgtized** [comment]
Fix was released in https://github.com/quil/quil/releases/tag/v4.3.1323.
